redis/conn.go

```diff
@@ -444,12 +458,12 @@ func (c *conn) writeArg(arg interface{}, argumentTypeOK bool) (err error) {
 type protocolError string
 
 func (pe protocolError) Error() string {
-	return fmt.Sprintf("redigo: %s (possible server error or unsupported concurrent read by application)", string(pe))
+	return fmt.Sprintf("redigo: %s (possible server error or unsupported concurrent r by application)", string(pe))
```

Comment:
accidental change? here and below

Reply:
https://github.com/filipecosta90/redigo/pull/1 to fix this 👍

Reply:
bump @filipecosta90

Reply:
sorry for the delay @platinummonkey =) on it

Reply:
solved in the latest commit